### ISSUE TITLE
Accessibility issues with new Calendar date/time picker

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#8193]
- Calendar date picker opens on focus. It is included in the tab index so it's not easy to skip. It is essentially impossible to skip using mobile screen reader as it has no close button. 
   - It should probably either not open on focus but only on click or down press (like Google Calendar) or it should open but should not get focused unless we press "down".
   -  On Mobile we use native date picker and date picker dropdown should not be shown at all.
 - Time picker is also shown on focus. It is also implemented as a dropdown which prevents us from isolating dropdowns like in #8164. It doesn't work to open it which mobile screen reader in most cases (at least with VoiceOver)
   - Should be reimplemented without modal component
   - Same suggestions as for calendar: either not show on focus or should not get focused
   - On Mobile we use native time picker and the dropdown should not be shown at all.
   - Make sure that it works with mobile screen reader: open native picker on click.

**Comment by marcosdiasalves** [comment]
Pay attention to this too: Event date picker does not support dark mode #7722

**Comment by andrehgdias** [comment]
Date picker and time picker has opposite behaviors.
Date picker opens only when pressing `arrow down`, but doesn't focus the "mini calendar"
Time picker opens only when pressing `return` and focus the first option instead of the selected value
Ideally both should work with boy keys and handle focus accordingly.

**Comment by murilopereirame** [comment]
> Date picker and time picker has opposite behaviors. Date picker opens only when pressing `arrow down`, but doesn't focus the "mini calendar" Time picker opens only when pressing `return` and focus the first option instead of the selected value Ideally both should work with boy keys and handle focus accordingly.

We agreed to open only with ```return``` that seems to be the standard
